utils/tf_solver.py

```diff
@@ -51,15 +51,15 @@ def set_objective(self, model_init_fn, dataset, normalization):
                 tf.keras.optimizers.schedules.ExponentialDecay(
                     value,
                     decay_rate=0.1,
-                    decay_steps=30,
+                    decay_steps=n_epochs//2,
```

Comment:
didn't we say `[n_epochs//2, 3*n_epochs//4]]` to get 2 steps?

Reply:
so the thinking was that in order to do a minimal PR, since it's not possible to have this specific schedule I would forget it. But I will actually try to change it.

Reply:
not that much of a change in the end haha!